src/EnergyPlus/EMSManager.cc

```diff
@@ -650,20 +576,20 @@ namespace EMSManager {
                                                                          cAlphaFieldNames,
                                                                          cNumericFieldNames);
                 UtilityRoutines::IsNameEmpty(state, cAlphaArgs(1), cCurrentModuleObject, ErrorsFound);
```

Comment:
This isn't necessary anymore.

Reply:
Verified as true:

    EnergyManagementSystem:Sensor,
      A1 , \field Name
           \required-field